From 858b957a03e717c76b6833d16ad108f2f1abc219 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 2 Nov 2010 09:59:09 -0500
Subject: [PATCH] arm: omap: devkit8000: for lcd use samsung_lte_panel

This was noticed by Robert Skretkowicz

Merged-to-2.6.37-git10-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/mach-omap2/board-devkit8000.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/board-devkit8000.c b/arch/arm/mach-omap2/board-devkit8000.c
index 00bb1fc..25d192c 100644
--- a/arch/arm/mach-omap2/board-devkit8000.c
+++ b/arch/arm/mach-omap2/board-devkit8000.c
@@ -159,7 +159,7 @@ static struct panel_generic_dpi_data lcd_panel = {
 static struct omap_dss_device devkit8000_lcd_device = {
 	.name                   = "lcd",
 	.type                   = OMAP_DISPLAY_TYPE_DPI,
-	.driver_name            = "generic_dpi_panel",
+	.driver_name            = "samsung_lte_panel",
 	.data			= &lcd_panel,
 	.phy.dpi.data_lines     = 24,
 };
-- 
1.7.1

